docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -68,7 +68,58 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
 . Click *Operators → Installed Operators* to verify that the {ProductShortName} Operator appears in the `openshift-mta` project with the status `Succeeded`.
 . Click the *{ProductShortName}* Operator.
 . Under *Provided APIs*, locate *Tackle*, and click *Create Instance*.
-. Review options - the default selections should be OK, but be sure to validate the system requirements for storage, memory, and cores - and then click *Create*.
++
+The *Create Tackle* window opens in *Form view*.
```

Comment:
@RichardHoch  
Does 'Form view' appear as the name of this screen? If not, I don't think 'view' should be in bold.

Reply:
@sbeskin-redhat Please take a look at the screen captures in https://issues.redhat.com/browse/TACKLE-914  -- they are why I bolded "view" but if you think that's unnecessary, let me know and I'll change the formatting.

Reply:
@RichardHoch 
No, 'view' should not be in bold.